Danish

```diff
@@ -0,0 +1,143 @@
+#define MSG_XSTEPS                          "Xsteps/mm"
+#define MSG_YSTEPS                          "Ysteps/mm"
+#define MSG_ZSTEPS                          "Zsteps/mm"
+#define MSG_ESTEPS                          "Esteps/mm"
+#define MSG_TEMPERATURE                     "Temperatur"
+#define MSG_MOTION                          "Motion"
+#define MSG_VOLUMETRIC                      "Filament"
+#define MSG_VOLUMETRIC_ENABLED		        "E in mm3"
+#define MSG_FILAMENT_SIZE_EXTRUDER_0        "Fil. Dia. 1"
+#define MSG_FILAMENT_SIZE_EXTRUDER_1        "Fil. Dia. 2"
+#define MSG_FILAMENT_SIZE_EXTRUDER_2        "Fil. Dia. 3"
+#define MSG_CONTRAST                        "LCD kontrast"
+#define MSG_STORE_EPROM                     "Gem memory"
+#define MSG_LOAD_EPROM                      "Hent memory"
+#define MSG_RESTORE_FAILSAFE                "Gendan failsafe"
+#define MSG_REFRESH                         "Genopfrisk"
+#define MSG_WATCH                           "Info skaerm"
+#define MSG_PREPARE                         "Forbered"
+#define MSG_TUNE                            "Tune"
+#define MSG_PAUSE_PRINT                     "Pause printet"
+#define MSG_RESUME_PRINT                    "Forsaet printet"
+#define MSG_STOP_PRINT                      "Stop printet"
+#define MSG_CARD_MENU                       "Print fra SD"
+#define MSG_NO_CARD                         "Intet SD kort"
+#define MSG_DWELL                           "Sleep..."
+#define MSG_USERWAIT                        "Venter paa bruger..."
+#define MSG_RESUMING                        "Forsaetter printet"
+#define MSG_PRINT_ABORTED                   "Print annuleret"
+#define MSG_NO_MOVE                         "No move."
+#define MSG_KILLED                          "KILLED. "
+#define MSG_STOPPED                         "STOPPED. "
+#define MSG_CONTROL_RETRACT                 "Tilbagetraek mm"
+#define MSG_CONTROL_RETRACT_SWAP            "Skift Re.mm"
+#define MSG_CONTROL_RETRACTF                "Tilbagetraek  V"
+#define MSG_CONTROL_RETRACT_ZLIFT           "Hop mm"
+#define MSG_CONTROL_RETRACT_RECOVER         "UnRet +mm"
+#define MSG_CONTROL_RETRACT_RECOVER_SWAP    "S UnRet+mm"
+#define MSG_CONTROL_RETRACT_RECOVERF        "UnRet  V"
+#define MSG_AUTORETRACT                     "AutoRetr."
+#define MSG_FILAMENTCHANGE                  "Skift filament"
+#define MSG_INIT_SDCARD                     "Init. SD card"
+#define MSG_CNG_SDCARD                      "Skift SD kort"
+#define MSG_ZPROBE_OUT                      "Z probe out. bed"
+#define MSG_POSITION_UNKNOWN                "Home X/Y foer Z"
+#define MSG_ZPROBE_ZOFFSET                  "Z Offset"
+#define MSG_BABYSTEP_X                      "Babystep X"
+#define MSG_BABYSTEP_Y                      "Babystep Y"
+#define MSG_BABYSTEP_Z                      "Babystep Z"
+#define MSG_ENDSTOP_ABORT                   "Endstop abort"
+
+#define MSG_RECTRACT                        "Tilbagetraek"
```

Comment:
This message has been gone for a while, so it looks like you picked a bad starting-point. You should start with something like `language_de.h` in the Development branch instead, copy it and name the copy `language_da.h` and do your translations there.

https://github.com/MarlinFirmware/Marlin/blob/Development/Marlin/language_de.h

Also, don't make your changes in a branch named `Development`. Create a new branch as a copy of `Development` (using Github or the Github app) and make your changes to the new branch. This will make it easier to deal with pull requests.